packages/loot-core/src/shared/months.ts

```diff
@@ -4,6 +4,26 @@ import memoizeOne from 'memoize-one';
 
 import * as Platform from '../client/platform';
 
+function isoWeekSettings(
+  date: Date,
+  type: string,
+  firstDayOfWeekIdx: number,
+): Date {
+  const checkNull = firstDayOfWeekIdx === undefined ? 1 : firstDayOfWeekIdx + 1;
+  const subNum =
```

Comment:
This quite possibly the ugliest code I've ever written (it does, however, work as intended). I couldn't figure out any other way to get the default ISO Day to change off of Monday. If anyone has suggestions I'd be happy to use them.

Reply:
@carkom why do we need it to be the ISO week? Can't we just use `d.startOfWeek(date, {weekStartsOn: firstDayOfWeekIdx)` and `d.endOfWeek(date, {weekStartsOn: firstDayOfWeekIdx})` ?

Being that we are working with a Date object by this point, my hope is that the function will work as we intend?

Reply:
Ahh! Yes! Thank you. I couldn't get out of my own head. Needed an outside set of eyes to diagnose it. Super elegant and easy solution. Thanks for the assist! 🧙‍♂️